orchestrator/settings.py

```diff
@@ -40,6 +40,7 @@ class AppSettings(BaseSettings):
         "ETag",
     ]
     ENVIRONMENT: str = "local"
+    EXECUTOR: str = "threadpool"
```

Comment:
Perhaps make this a string enumeration ?

Reply:
💯

Reply:
Not 100 % sure if that is very useful for the value of environment variables, but added.